actions/workflows/npm_publish_version.yaml

```diff
@@ -0,0 +1,43 @@
+---
+  chain:
+    -
+      name: "get_build_server"
+      ref: "linux.dig"
+      params:
+        hostname: "st2-build-slave-ubuntu.service.consul"
+        rand: true
+        count: 1
+      publish:
+        build_server: "{{get_build_server.result[0]}}"
+      on-success: "clone_repo"
+    -
+      name: "clone_repo"
+      ref: "st2cd.git_clone"
+      params:
+        hosts: "{{build_server}}"
+        repo: "{{repo}}"
+        branch: "{{branch}}"
+        target: "{{repo_target}}"
+      publish:
+        repodir: "{{clone_repo[build_server].stdout}}"
+      on-success: "npm_publish"
+    -
+      name: "npm_publish"
+      ref: "core.remote"
+      params:
+        hosts: "{{build_server}}"
+        cmd: "cd {{repodir}} && npm version {{update_type}} && npm publish"
```

Comment:
I assume we want to increment the version ourselves since `npm version` is naive and doesn't know which version you want to increment (major / minor / patch).